include/ipc/ipc_service.h

```diff
@@ -165,7 +167,7 @@ int ipc_service_register_endpoint(const struct device *instance,
  *  @retval -EBADMSG when the message is invalid.
  *  @retval -EBUSY when the instance is busy.
  *
- *  @retval 0 on success.
+ *  @retval number of bytes it has sent.
```

Comment:
nit: `@retval` syntax is `@retval VALUE DESCRIPTION`, so maybe:

```suggestion
 *  @retval bytes Number of bytes it has sent.
```

Reply:
the least number of lines I change the most number of errors I make :D